src/main/scala/org/broadinstitute/dsde/workbench/sam/api/OldUserRoutes.scala

```diff
@@ -151,33 +150,4 @@ trait UserRoutes extends SamUserDirectives with SamRequestContextDirectives {
         }
       }
     }
-
-  def apiUserRoutes(samUser: SamUser, samRequestContext: SamRequestContext): server.Route = pathPrefix("users") {
```

Comment:
are we just totally removing these? should there be a corresponding removal in swagger?